src/cpp/proxy/wscontrolmanager.h

```diff
@@ -26,6 +26,7 @@
 
 #include <QObject>
 #include "packet/wscontrolpacket.h"
+#include <memory>
```

Comment:
nit: move above `#include <QObject>`

(our include order is typically: std, lib, local)